### ISSUE TITLE
t/end_to_end.t fails with Net::AMQP::RabbitMQ 1.000000

### ISSUE THREAD

**Issue by abraxxa** [abraxxa/Log-Log4perl-Appender-RabbitMQ#2]
```
t/end_to_end.t ............ 1/5 
# Failed test 'info message sent to Rabbit with proper format'
# at t/end_to_end.t line 56.
# Structures begin differing at:
# $got->{props} = HASH(0x2dedc00)
# $expected->{props} = undef
# Failed test 'warn message sent to Rabbit with proper format'
# at t/end_to_end.t line 69.
# Structures begin differing at:
# $got->{props} = HASH(0x2f58418)
# $expected->{props} = undef
# Looks like you failed 2 tests of 5.

t/end_to_end.t ............ Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/5 subtests
```

props is now an empty hash and redelivered  => 0 is included in the returned hashref as well.

**Comment by abraxxa** [comment]
Hi Trevor,
if you're interested in handing over maintenance for this module because you don't have time for it any more please contact me.

**Comment by bundacia** [comment]
@abraxxa, yeah, I haven't worked with perl for a while now. I'd be happy to hand over maintenance. What's the best way to do that?

**Comment by abraxxa** [comment]
Sorry to hear that!
Give maintainer or firstcome to me on PAUSE to be able to release new versions and move the github repo should be sufficient.

**Comment by abraxxa** [comment]
Just found out that TLS isn't supported currently either as various `ssl` options aren't passed on to Net::AMQP::RabbitMQ.
Please give me maint as soon as possible so I can work on that.
Thanks!

**Comment by bundacia** [comment]
Ok, I initiated the github ownership transfer to you @abraxxa. I attempted to start a transfer to you in pause but I may have done that wrong. I gave up ownership but there's didn't seem to be a way to transfer ownership to you. Right now I'm listed as co-maintainer and there's no owner.

![pause__peek_perms](https://cloud.githubusercontent.com/assets/189699/23169178/c3c3ecb0-f818-11e6-890d-bcf4c87cf157.png)

**Comment by abraxxa** [comment]
Thanks!
Did you use 'Transfer primary maintainership status' in PAUSE or 'Give up primary maintainership status'?

**Comment by bundacia** [comment]
I'm pretty sure I used "Transfer primary maintainership status to somebody else (you become co-maintainer)" unless I clicked the wrong button on accident. I was surprised when it never asked me for the username of a new owner/maintainer.

**Comment by abraxxa** [comment]
mst checked it and you've pressed the wrong button but he will fix it.
Do you still want co-maintainership?

**Comment by abraxxa** [comment]
Can you please do one last commit (pull-request) for the copyright_holder change in dist.ini or do you want to keep it? Thanks!

**Comment by bundacia** [comment]
You can change the copyright (now that the repo transferred I don't think I have write access anymore). I also don't need co-maintainership.

**Comment by abraxxa** [comment]
fixed in version 0.200001 by https://github.com/abraxxa/Log-Log4perl-Appender-RabbitMQ/commit/db9786501570ab6d31d97ddbfc1a92dd3e5e46c4

**Comment by abraxxa** [comment]
Thanks Trevor!
